tmd/Tree/Tree.py

```diff
@@ -25,8 +28,8 @@ class Tree(object):
     from tmd.Tree.methods import get_multifurcations
     from tmd.Tree.methods import get_terminations
 
-    def __init__(self, x=_np.array([]), y=_np.array([]), z=_np.array([]),
-                 d=_np.array([]), t=_np.array([]), p=_np.array([])):
+    def __init__(self, x=np.array([]), y=np.array([]), z=np.array([]),
```

Comment:
every tree will share the same arrays.

Reply:
Again I only modified the _np to np. It is not true that all trees have the same coordinates. I don't understand this comment because all test pass (which I think indicates that this comment is not correct :/ )